TAScheduler/AcceptanceTests/test_edit_section.py

```diff
@@ -0,0 +1,102 @@
+# PBI #11: Edit Section
+from datetime import datetime
+
+from django.test import TestCase, Client
+
+from TAScheduler.models import Section, Course, Lecture, Lab
+
+
+# AC1] - Successful Edit (>0 sections, valid changes, no missing)
+class SuccessEdit(TestCase):
+    def setUp(self):
+        self.client = Client()
+        self.courseList = list()  # holding list of test courses so easier to call later
+        self.secList = list()
+        # Creating 3 Courses: hardcoded "numSec","modality","credits"
+        for i in [1, 2, 3]:
+            testCourse = Course(course_id=i, semester="semester" + str(i), name="name" + str(i), num_of_sections=2,
+                                modality="Remote", credts=3, )
+            testCourse.save()
+            self.courseList = self.courseList.append(testCourse)
+        # Creating 3 Sections: hardcoded "meetingTime"
+        for i in [1, 2, 3]:
+            testSection = Section(section_id=i, course=self.courseList[i - 1], location="location" + str(i),
+                                  meeting_time=datetime(2023, 1, 1, 1, 1, 1))
+            testSection.save()
+            self.secList = self.secList.append(testSection)
+
+    # 1] Database reflects our input
+    def test_DatabaseEditChange(self):
+        secChanges = {"location": "Naboo"}
+        resp = self.client.post("home/managesection/edit", data={"section": self.secList[0], "edit": secChanges})
+        dbEditedSect = Section.objects.filter(section_id=self.secList[0].section_id)
+        self.assertEquals(dbEditedSect["location"], "Naboo")
+
+    # 2] Confirmation message
+    def test_ConfirmationMessage(self):
+        secChanges = {"location": "Naboo"}
```

Comment:
Again with the testing of a confirmation message if we saw it worked in the last test why are we checking this?

Reply:
The messages would be different - we can't have someone edit an account and have our views return "successfully created account." This is a part of the PBI + Rock tested for messages in his example. Thanks for your consideration!